app/components/SettingsCell.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { View, Text, StyleSheet, TouchableOpacity, Switch } from 'react-native'
+import { colors } from '../config'
+import Divider from './Divider'
+
+const { cellTitle } = colors
+
+const SettingsCell = ({
+  title,
+  onCellPress,
+  onSwitchValueChange,
+  titleNumLines,
+  icon,
+}) => {
+  const cellPressed = () => (onCellPress ? onCellPress() : null)
+  const switchValueChanged = (value) =>
+    onSwitchValueChange ? onSwitchValueChange(value) : null
+
+  const { titleText, wrapper, innerContent, icon } = styles
+  return (
+    <TouchableOpacity
+      onPress={cellPressed}
+      activeOpacity={cellPressed == null ? 1 : undefined}
+    >
+      <View style={wrapper}>
+        <Text style={titleText} numberOfLines={titleNumLines}>
+          {title}
+        </Text>
+        <View style={innerContent}>
+          {onCellPress == null ? (
+            <Switch onValueChange={switchValueChanged} />
+          ) : (
+            icon
```

Comment:
Sorry this isn't what I meant. I meant a prop `icon` that is a string we can use to render the icon so it would look like <Icon name={icon} />.

Or do you expect icon here to be a component?

Reply:
@MDHayes ok as i thought that you want me to add icon name but what happens if on some of other component we want to use different style for icon like its size or color?

Reply:
or we can pass iconStyle and iconName as as prop?

Reply:
I'm happy with a component being passed in, if that's the approach you prefer but you need to add a component with the chevron in this PR do the feature is complete

If using an `Icon` from `react-native-vector-icons` it has props for size, color and style, you can pass them on/set defaults. I think this approach is fine.

Reply:
> If using an `Icon` from `react-native-vector-icons` it has props for size, color and style, you can pass them on/set defaults. I think this approach is fine.

yeah i also think this will be fine ok i will do desired changes.